Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift

```diff
@@ -274,6 +301,47 @@ class SafeSettingsViewController: LoadableViewController, UITableViewDelegate, U
         }
     }
 
+    func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        guard isValid(path: indexPath), safe != nil && !safeOwners.isEmpty else { return nil }
+
+        let item = sections[indexPath.section].items[indexPath.row]
+        switch item {
+        case Section.OwnerAddresses.ownerInfo(let info):
+            guard let ownerIndex = (safeOwners.firstIndex { $0.address == info.address }) else { return nil }
+            let prevOwner = safeOwners.before(ownerIndex)
+            let deleteAction = UIContextualAction(style: .destructive, title : "Remove") {
+                [unowned self] _, _, completion in
+                self.remove(owner: info.address)
+                completion(true)
+            }
+            deleteAction.backgroundColor = .error
+
+            let replaceAction = UIContextualAction(style: .normal, title : "Replace") {
+                [unowned self] _, _, completion in
+                // TODO: Display replace owner flow
+                completion(true)
+            }
+            replaceAction.backgroundColor = .tertiaryLabel
+
+            return UISwipeActionsConfiguration(actions: [deleteAction, replaceAction])
+        default:
+            return nil
+        }
+    }
+
+    func remove(owner: Address) {
+        guard let navigationController = navigationController else {
+            return
+        }
+
+        removeOwnerFlow = RemoveOwnerFlow(
+            owner: owner,
+            safe: safe!,
+            navigationController: navigationController) { [unowned self] skippedTxDetails in
```

Comment:
you need to remove the reference to 'removeOwnerFlow' in the completion block because otheriwse it retains reference to a navigation controller

Reply:
Another comment: the variable in the completion is not used.